src/test/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingDeprecatedCheckTest.java

```diff
@@ -168,4 +168,13 @@ public void testPackageInfo() throws Exception {
         verify(checkConfig, getPath("package-info.java"), expected);
     }
 
+    @Test
+    public void testPackageInfoBelowComment() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(MissingDeprecatedCheck.class);
+
+        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
```

Comment:
Please also add a UT with issue but where it will produce a violation to show check functionality still works in this situation.

Reply:
The file I am passing to verify does have two violations, but they are from SummaryJavadoc.  Should I add some code to trigger violations from `MissingDeprecatedCheck`, i.e. `annotation/messges.properties`?

Reply:
>  Should I add some code to trigger violations from MissingDeprecatedCheck

Yes, this is what I am looking for. This test is only for MissingDeprecatedCheck so all examples should focus around it. This is what is being reported to us in issue.

Reply:
Please let me know if there's any other checks I should add, it didn't seem like any others would apply to packages in `MissingDeprecatedCheck`.